app/controllers/container_dashboard_controller.rb

```diff
@@ -25,10 +27,18 @@ def data_live
     render :json => collect_live_data(params[:id], params[:query])
   end
 
+  def title
+    "container_dashboard"
+  end
+
   private
 
+  def self.session_key_prefix
+    "container_dashboard"
+  end
+
   def get_session_data
-    @layout = "container_dashboard"
```

Comment:
Ok, if all that `get_session_data` does here is to set `@layout` and sets it to a constant, then maybe this controller is not a good candidate for `GenericSessionMixin`...

Maybe you could just keep `get_session_data` and remove `set_session_data`...